proto/cosmos/offchain/v1alpha1/offchain.proto

```diff
@@ -0,0 +1,15 @@
+syntax="proto3";
+
+package cosmos.offchain.v1alpha1;
```

Comment:
how about storing this file in `proto/cosmos/auth/v1beta1/offchain-sig/`?

Reply:
After the meeting today, I suggest to move this to `proto/cosmos/offchain/adr038` : 
* `offchain` - a place where we group offchain use-cases
* `adr038` - name of this use-case - unless you have better idea?